test/e2e/validation_test.go

```diff
@@ -136,6 +136,30 @@ func (tc *testContext) getInstanceIP(instanceID string) (string, error) {
 	return "", errors.New("unable to find Windows Worker nodes")
 }
 
+// getNode returns a pointer to the node object associated with the instance ID provided
+func (tc *testContext) getNode(instanceID string) (*corev1.Node, error) {
```

Comment:
Can't this be merged with `tc.GetInstanceIP`? As in there is common code between the two functions as far as finding the node object goes.

Reply:
Not sure what you have done. What I was thinking is that your would call this function in `GetInstanceIP`.